packages/styles/scss/patterns/blocks/content-block-simple/_content-block-simple.scss

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+
+@mixin content-block-simple {
+  .#{$prefix}--content-block-simple {
+    &__content {
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    .#{$prefix}--content-item {
+      margin-bottom: $carbon--spacing-07;
+
+      &__heading {
+        margin-top: 0;
+      }
+
+      p {
+        margin-bottom: $carbon--spacing-07;
```

Comment:
I'll change to `06` but I purposely didn't change it in `contentItem` because i don't want to create issues in other patterns/groups that are using it. We can make this an issue to fix